src/components/Footer/Footer.js

```diff
@@ -133,6 +232,7 @@ const Footer = ({
           </View>
         </TouchableOpacity>
       </View>
+      <BottomSheetMenu refSheet={refSheet} dataSheet={dataSheet} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BottomSheetMenu` component is rendered here with the `refSheet` and `dataSheet` props. Ensure that this component is implemented correctly and that it receives and handles these props as expected.



<!-- This is an auto-generated comment by OSS CodeRabbit -->